### PR TITLE
Add minutes for 2017-09-28 meeting

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Minutes for this week's meeting.

### CODE REVIEW

File: organization/_posts/2017-09-28-startup.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "HSF Weekly Meeting #110, September 28, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #110, September 28, 2017
+
+#### *Present*: Graeme Stewart, Dario Menasce, Giacomo Tenaglia, Justin Chiu, Ian Bird, Sergei Gleyzer, Pere Mato, Torre Wenaus, Marko Petric, Pete Elmer, Michel Jouvin, John Harvey, Stefan Rosier, Rob Kutschke
```
Comment (hegner) [inline organization/_posts/2017-09-28-startup.md]: Stefan *Roiser*
and I seem missing ;-)

File: organization/_posts/2017-09-28-startup.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        - Otherwise recommended licenses remain GPLv3, LGPLv3 and Apache2.
+    - Intellectual property is split 50 / 50 by default, and negotiated in case of developments “from scratch”. Again for add-ons to existing software (Oracle and Siemens mainly) the IP is fully with the company.
+- Update from CERN KT:
+    - “Software dissemination policy” document released
+    - Will work with CERN IT on a Contributor License Agreement (CLA) for CERN open source projects. The main issue to be decided is about (C) transfer or not: there will probably be two versions/templates, one for each case. KT lawyers are working on the legal side, IT is working on a generic “how to open source my code” page which will include the CLA. ETA November 2017.
+    - CERN KT is helping CERN projects to do “due diligence” on their FOSS code in order to change license (especially for a couple of CERN IT projects where they contact authors), but would prefer not to have to do it for HSF code. Nevertheless they encourage undertaking the process. One important thing is to check who owns the (C), which institutes, etc.
+
+### Training
+- GridKa feedback about their use of WikiToLearn: none
```
Comment (hegner) [inline organization/_posts/2017-09-28-startup.md]: No GridKa feedback about their experience yet.

**Comment by hegner** [comment]
You may want to look into the HTML-proofer problems again :-(

**Comment by hegner** [comment]
Thanks. Ignoring URL validator problems in travis as they seem to be caused by server instabilities and not by invalid links.

**Comment by graeme-a-stewart** [comment]
Thanks. I agree the remote site problems look temporary, e.g., the portage site is having DB connection issues.

**Pull request merged**